examples-staging/ecommerce/schemas/Product.ts

```diff
@@ -44,8 +44,14 @@ export const Product = list({
     price: integer(),
     user: relationship({
       ref: 'User.products',
-      defaultValue: ({ context }) =>
-        context.session?.itemId ? { connect: { id: context.session?.itemId } } : null,
+      hooks: {
+        resolveInput({ operation, resolvedData, context }) {
```

Comment:
```suggestion
        resolveInput({ operation, resolvedData, context }) {
          // Default to the currently logged in user on create.
```

Took my brain too long to work out the semantics of this, so adding a comment to save the next poor soul.